HACKING.md

```diff
@@ -260,7 +267,16 @@ The easiest way to set up a juju controller is inside a
 remove controllers as necessary and provides a way to have multiple controllers
 running at once (with different Juju versions if needed).
 
-If a Kubernetes controller is required you can follow the [Multipass and Kubernetes](./docs/multipass-microk8s.md) instructions.
+There are three main types of deployment:
+- [Juju controller](#juju-controller)
+- [Juju with Kubernetes](./docs/multipass-microk8s.md)
+- [JIMM controller](#jimm-controller)
+
+### Kubernetes controller
+
+If a Kubernetes controller is required you can follow the  instructions.
```

Comment:
Should there be a link to instructions here as well?

Reply:
Oops, I moved the link but forgot to remove this section. Thanks.